tests/factories.py

```diff
@@ -0,0 +1,58 @@
+import factory
```

Comment:
Oh I think I'm missing `factory-boy`...updating the requirements should fix this!